### ISSUE TITLE
Ticketmaster webpage third-party content causing NSURLErrorDomain -1002

### ISSUE THREAD

**Issue by digitaldavenyc** [react-native-community/react-native-webview#566]
**Bug description:**
Some sites have content that does not follow proper web standards and are causing our component to fail hard. There needs to be a way to let these errors fail silently so the entire page doesn't crash.

I have `Allow Arbitrary Loads` & `NSAllowsArbitraryLoadsInWebContent` enabled in our Info.plist file 

```
<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<true/>
		<key>NSAllowsArbitraryLoadsInWebContent</key>
		<true/>
	</dict>
```

I also have set `mixedContentMode="always"`

**To Reproduce:**
Loading any Ticketmaster.com ticketing page will initially load and as secondary content (JS, images, etc...) loads an error will be thrown.
For ex: https://www1.ticketmaster.com/billy-joel-billys-70th-birthday-new-york-new-york-05-09-2019/event/3B00554F06438807

The URL will throw a NSURLErrorDomain with error code: -1002 (unsupported URL)

**Expected behavior:**
Webview does not throw error on webpage content or secondary url requests, should be able to fail gracefully on nested content instead of a hard failure.

**Comment by asrytis** [comment]
This also happens with any Medium.com article. Did you ever find a solution?

**Comment by asrytis** [comment]
No need to patch the native code. Here's how to solve it:
```
<WebView originWhitelist={['http://*', 'https://*']} ... />
```

**Comment by groovemaps** [comment]
@asrytis I have not found a solution to this but I will certainly try this. I would rather not have to enter a huge list of URL schemes and might just add a return, not sure what that will do since I am not an iOS developer either.

**Comment by groovemaps** [comment]
@asrytis are you sure that will work? I have the following set in our code and still get the error:  
<WebView originWhitelist={['*']} />

**Comment by asrytis** [comment]
This is the reason you get the error because you allow WebView to load any scheme. It’s probably attempting to load some weird resource like the ‘medium://‘ link in my example above.

**Comment by groovemaps** [comment]
@asrytis Interesting ... I thought it was because `http` content was being loaded in a `https` url

**Comment by Titozzz** [comment]
Can I close this ?

**Comment by digitaldavenyc** [comment]
@Titozzz For now yes, we've tested on several ticketmaster URLs (and around 50 others) and the issue seems to be curbed. We may need to reopen since it wasn't really clear what was causing the error and it could return. Thanks @asrytis for the fix!

**Comment by Overtorment** [comment]
if anyone is strugling with this on iOS, what solved it for me was:


```
<dict>
    <key>LSApplicationQueriesSchemes</key>
    <array>
        <string>YOUR SCHEME</string>
    </array>
</dict>
```

in `Info.plist`

**Comment by whitefang57** [comment]
@Overtorment does the scheme need to be all caps, or how does the letter case affect it?

**Comment by Overtorment** [comment]
@whitefang57 no difference i think, but i have it lower case

**Comment by LiliyaStefanova** [comment]
I have tried all of the suggestions above and still no luck - I am getting the same error. 
Further I tried to encode the URL which is being rendered in the webview (in case of special characters, though I don't have any query params) and to control the rendering of the url through onShouldStartLoadWithRequest, but it does not get detected before the error is triggered. 
Should I raise another issue to discuss this further?
